pkgs/shells/fish/plugins/puffer.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildFishPlugin, fetchFromGitHub }:
+
+buildFishPlugin rec {
+  pname = "puffer";
+  version = "unstable-2021-05-21";
+
+  src = fetchFromGitHub {
+    owner = "nickeb96";
+    repo = "puffer-fish";
+    rev = "df333fff5130ef8bf153c9bafbf0661534f81d9c";
+    sha256 = "sha256-VtFrRzI476Hkutwwgkkc9hoiCma6Xyknm7xHeghrLxo=";
```

Comment:
nit
```suggestion
    hash = "sha256-VtFrRzI476Hkutwwgkkc9hoiCma6Xyknm7xHeghrLxo=";
```

Reply:
why?
from the [Nixpkgs Manual](https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub):
> Finally, sha256 corresponds to the hash of the extracted directory. Again, other hash algorithms are also available, but sha256 is currently preferred.

Reply:
it looks like they have done a poor job of communicating this, but NixOS/rfcs#131 lays out where things are going. you will also notice the use of `hash` throughout the linked nixpkgs manual in other sections

as well, I listed this as a nit, since it does work, so you are welcome to ignore this suggestion.

Reply:
my bro, it is not merged yet

Reply:
yeah, that change is about enforcement, I was having trouble finding something merged and authoritative, but the community has been migrating for a while and it is very much a new standard.

I called it out to highlight that it is changing, not to require anything, so if you are opposed to making the change you are welcome to leave it alone.

Reply:
i mean if it works the same it doesn't really matter but i would only start to implement rfcs that have been merged and thus accepted by the  community

Reply:
yep, it will continue to still work, but it is suggested that we mention it during reviews: https://discourse.nixos.org/t/20493

the rfc is just about enforcing it.